roles/host_setup/templates/cloud-init/connect-runner.sh.j2

```diff
@@ -123,7 +123,7 @@ register_github_runner() {
         ./config.sh --url "https://github.com/$GITHUB_URL" \
             --token "$GITHUB_RUNNER_TOKEN" \
             --name "$RUNNER_NAME" \
-            --runnergroup "default" \
+            --runnergroup "Zededa Inc. - Self Hosted Runners" \
```

Comment:
I'd prefer this to be a parameter, which we specify in inventory.yml like we do with [tags](https://github.com/uncleDecart/ansible-runners-infra/blob/5f487efc52224e14037dc902d717b4709e529466/inventory.yml#L20)

so this would change to 

```
--runnergroup {{ vms.global.github_runner.group }}
```